src/sass/modules/_m-schemaform.scss

```diff
@@ -231,3 +231,41 @@ label {
     padding-left: 25px !important;
   }
 }
+
+//duplicated from disability-benefits.scss
```

Comment:
Thanks @jbalboni, created [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/3193).